public/grpc-client.js

```diff
@@ -48,6 +48,7 @@ function getMacaroonCreds(lndSettingsDir, network) {
 
 module.exports.init = async function({
   ipcMain,
+  lndIP,
```

Comment:
Set default value `lndIP = 'localhost'`

Reply:
good point, fixed. could do the same with `lndPort`